src/main/java/uk/gov/digital/ho/hocs/audit/service/AuditEventService.java

```diff
@@ -24,76 +27,57 @@ public class AuditEventService {
 
     private final AuditRepository auditRepository;
 
-    private final JsonValidator jsonValidator;
+    private final EntityManager entityManager;
 
     @Autowired
     public AuditEventService(AuditRepository auditRepository,
-                             JsonValidator jsonValidator) {
+                             EntityManager entityManager) {
         this.auditRepository = auditRepository;
-        this.jsonValidator = jsonValidator;
+        this.entityManager = entityManager;
     }
 
     public AuditEvent createAudit(String correlationID, String raisingService, String auditPayload, String namespace, LocalDateTime auditTimestamp, String type, String userID) {
         return createAudit(null, null, correlationID, raisingService, auditPayload, namespace, auditTimestamp, type, userID);
     }
 
     public AuditEvent createAudit(UUID caseUUID, UUID stageUUID, String correlationID, String raisingService, String auditPayload, String namespace, LocalDateTime auditTimestamp, String type, String userID) {
-        String validAuditPayload = jsonValidator.validateAuditPayload(correlationID, raisingService, auditPayload, namespace, auditTimestamp, type, userID);
-        AuditEvent auditEvent = new AuditEvent(caseUUID, stageUUID, correlationID, raisingService, validAuditPayload, namespace, auditTimestamp, type, userID);
-        validateNotNull(auditEvent);
+        AuditEvent auditEvent = new AuditEvent(caseUUID, stageUUID, correlationID, raisingService, auditPayload, namespace, auditTimestamp, type, userID);
         auditRepository.save(auditEvent);
-        log.debug("Created Audit: UUID: {}, CaseUUID: {}, StageUUID: {}, Correlation ID: {}, Raised by: {}, By user: {}, at timestamp: {}, event {}",
-                auditEvent.getUuid(),
-                auditEvent.getCaseUUID(),
-                auditEvent.getStageUUID(),
-                auditEvent.getCorrelationID(),
-                auditEvent.getRaisingService(),
-                auditEvent.getUserID(),
-                auditEvent.getAuditTimestamp(), value(EVENT, AUDIT_EVENT_CREATED));
+        log.debug("Created Audit: UUID: {} at timestamp: {}", auditEvent.getUuid(), auditEvent.getAuditTimestamp());
         return auditEvent;
     }
 
     public Integer deleteCaseAudit(UUID caseUUID, Boolean deleted){
-        log.debug("Case {} setting deleted to {}", caseUUID, deleted);
         List<AuditEvent> audits = auditRepository.findAuditDataByCaseUUID(caseUUID);
```

Comment:
Instead of deleting each, can we do this in bulk. Separate to this change though...

Reply:
We could do a native query and have it do a 

```
UPDATE audit_data 
SET deleted = true 
WHERE case_uuid = 'x'
```
Not sure its worth it - there is no clever JPA way.